### PR TITLE
Issue #3024698 by volkerk, chr.fritsch, daniel.bosen, alexpott: Prepare Thunder for Drupal 8.7

### PR DESCRIPTION
**Pull request by ol0lll**
Make sure these boxes are checked before submitting your pull request - thank you!

- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/develop/docs/development.md#how-to-run-the-tests))
- [x] Necessary update hooks are provided.
- [x] User roles have correct access for new introduced permission.
- [x] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [x] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: config/optional/core.entity_form_display.node.article.default.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ dependencies:
     - scheduler
     - scheduler_content_moderation_integration
     - select2
-    - thunder
```
Comment (chrfritsch) [inline config/optional/core.entity_form_display.node.article.default.yml:24]: A config export removes the dependency on thunder. But why? The moderation_state field still uses the thunder_moderation_state_default widget, which is placed in Drupal\thunder\Plugin\Field\FieldWidget

**Comment by chrfritsch** [comment]
MetaInformationTest fails because of https://www.drupal.org/project/drupal/issues/3025250

**Comment by chrfritsch** [comment]
So we are almost there. The following things needs to be done before we can merge this:

- [ ] Use Drupal 8.6 for drush when we merge before 8.7 is released
- [ ] Finish https://github.com/BurdaMagazinOrg/theme-thunder-admin/pull/186 in a way that it works for 8.6 and 8.7
- [x] Try to get the FB patch committed

**Comment by chrfritsch** [comment]
https://www.drupal.org/project/access_unpublished/issues/3028106 appears while activating jsonapi

**Review by dbosen — APPROVED** [review]
We should merge this now, to be able to finish up the admin theme work.
After the theme has been updated, we have to open a new PR changing the THUNDER_ADMIN_BRANCH to 8.x-2.x.

**Pull request merged**